### ISSUE TITLE
Pino wont remember that I hide the menu and toolbar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#7]
```
When I hide the menu and toolbar everything works fine. But when I restart
Pino the menu and toolbar are visible again.

```

Original issue reported on code.google.com by `58desi...@gmail.com` on 14 Dec 2009 at 12:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 14 Dec 2009 at 11:19
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by fe320a0c24 revision.
```

Original comment by `tro...@gmail.com` on 14 Dec 2009 at 3:04
- Changed state: **Fixed**
